app/components/bali/breadcrumb/index.scss

```diff
@@ -0,0 +1,2 @@
+.breadcrumb-component {
+}
```

Comment:
If we do not have any style, do we need this file?

Reply:
we don't, i'll delete